android/src/main/java/com/thebluealliance/androidclient/viewmodels/ViewModelRenderer.java

```diff
@@ -0,0 +1,11 @@
+package com.thebluealliance.androidclient.viewmodels;
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+import android.support.annotation.WorkerThread;
+
+public interface ViewModelRenderer<VIEWMODEL, ARGS> {
+
+    @WorkerThread
+    @Nullable VIEWMODEL renderToViewModel(Context context, @Nullable ARGS args);
```

Comment:
Would we rather enforce a parent class/interface for all the ViewModels? That way, we at least know the base class (rather than just `Object`), which may be useful for handling more generic approaches later? 

That way, the return type of this class just has to be `ParentViewModel`